data_collection/gazette/spiders/mg/mg_piumhi.py

```diff
@@ -0,0 +1,47 @@
+from datetime import date, datetime
+
+import scrapy
+
+from gazette.items import Gazette
+from gazette.spiders.base import BaseGazetteSpider
+
+
+# Create a class for the spider
+class MgPiumhiSpider(BaseGazetteSpider):
+    name = "mg_piumhi"
+    TERRITORY_ID = "3151503"
+    allowed_domains = ["prefeiturapiumhi.mg.gov.br"]
+    base_url = "https://diario-oficial.prefeiturapiumhi.mg.gov.br/"
+    start_date = date(2023, 9, 11)
+
+    def start_requests(self):
+        yield scrapy.Request(
+            url=f"{self.base_url}",
+        )
```

Comment:
```suggestion
    start_date = date(2023, 9, 11)
    start_urls = ["https://diario-oficial.prefeiturapiumhi.mg.gov.br/"]

```

Reply:
**sugestão** Usamos o `start_requests` quando é necessário acrescentar lógica na geração dos requests (como por exemplo, diferentes requests para uma faixa de datas, etc.). Quando queremos simplesmente abrir uma URL, usamos o start_urls` ([referência](https://docs.scrapy.org/en/latest/intro/tutorial.html#a-shortcut-to-the-start-requests-method))